import java.sql.*;

public class InserData {
    public static void main(String[] args) throws SQLException {
        Connection myConn = null;
        Statement myStmt = null;
        ResultSet myRs = null;

        try {
            // 1. Get a connection to database
            myConn = DriverManager.getConnection("jdbc:mysql://localhost:3307/demo", "root" , "root");

            // 2. Create a statement
            myStmt = myConn.createStatement();

            // 3. Execute SQL query
            myStmt.executeUpdate(
                    "INSERT INTO employees " +
                            "(id, last_name, first_name, email, department, salary) " +
                            "VALUES (NULL, 'Carvalho', 'Ricardo', 'ricardo.teste@foo.com', 'Engineering', 90000.00)");

            myRs = myStmt.executeQuery("select * from employees");

            // 4. Process the result set
            while (myRs.next()) {
                System.out.println(myRs.getString("last_name") + ", " + myRs.getString("first_name"));
            }
        }
        catch (Exception exc) {
            exc.printStackTrace();
        }
        finally {
            if (myRs != null) {
                myRs.close();
            }

            if (myStmt != null) {
                myStmt.close();
            }

            if (myConn != null) {
                myConn.close();
            }
        }
    }

}
